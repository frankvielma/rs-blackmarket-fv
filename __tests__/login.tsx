import { Image } from 'next/Image'
import { render } from '@testing-library/react'
import Login from '@/app/login/page'

it('renders homepage unchanged', () => {
  const { container } = render(<Login />)
  expect(container).toMatchSnapshot()
})
