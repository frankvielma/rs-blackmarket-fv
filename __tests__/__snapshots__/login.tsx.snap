// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="flex h-[100vh] flex-col bg-[url('/images/bm_wallpaper.jpg')] bg-cover bg-center bg-no-repeat px-6 py-16 text-base text-slate-950 md:px-28 className"
  >
    <div
      class="h-[366px] max-w-[328px] rounded-lg border-black bg-white md:h-[425px] md:max-w-[360px]"
    >
      <div
        class="pb-9 pt-[40px] md:p-[47px] md:pb-12"
      >
        <img
          alt="Black Market Logo"
          class="m-auto"
          data-nimg="1"
          decoding="async"
          height="173"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fbm_logo.png&w=384&q=75"
          srcset="/_next/image?url=%2Fimages%2Fbm_logo.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fbm_logo.png&w=384&q=75 2x"
          style="color: transparent;"
          width="173"
        />
      </div>
      <form>
        <div
          class="flex flex-col"
        >
          <label
            class="px-[34px] text-sm text-slate-950 md:text-base"
            for="email"
          >
            Email
          </label>
          <input
            class="custom-input custom-focus custom-active md:h-[44px]"
            name="email"
            placeholder="Type your email"
            type="email"
          />
        </div>
        <div
          class="relative my-4"
        >
          <label
            class="px-[34px] text-sm text-slate-950 md:text-base"
            for="password"
          >
            Password
          </label>
          <div
            class="flex flex-col"
          >
            <input
              class="custom-input custom-focus custom-active md:h-[44px]"
              name="password"
              placeholder="Type your password"
              type="password"
            />
            <div
              class="absolute right-12 pt-2 md:pt-4"
            >
              <img
                alt=""
                class="w-4"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="/icons/visibility_off.svg"
                style="color: transparent;"
                width="16"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col"
        >
          <input
            class="custom-focus mx-[34px] h-[44px] w-auto rounded-md border border-slate-950 bg-slate-950 font-bold text-white"
            type="submit"
            value="Log in"
          />
        </div>
      </form>
      <div
        class="pt-4 text-center md:pt-9"
      >
        <a
          class="custom-focus mt-10 font-bold text-blue-600 hover:underline "
          href="/forgot-password"
        >
          I forgot my password.
        </a>
      </div>
    </div>
    <div
      class="mt-4 h-[121px] max-w-[328px] rounded-lg border-black bg-white md:h-[130px] md:max-w-[360px]"
    >
      <div
        class="mt-5 text-center"
      >
        Don't have an account?
      </div>
      <div
        class="custom-focus mx-[34px] mt-4 h-[44px] w-auto rounded-md border border-slate-950 bg-white p-2 text-center font-bold text-black "
      >
        <button
          class="focus:outline-none"
        >
          Sign up
        </button>
      </div>
    </div>
  </main>
</div>
`;
